requirements/common.txt

```diff
@@ -97,4 +97,7 @@ certifi==2016.9.26 --hash=sha256:a60073d7bb5cb01671877efda875e8dab755b98baedfbb9
 slugid==1.0.7 --hash=sha256:6dab3c7eef0bb423fb54cb7752e0f466ddd0ee495b78b763be60e8a27f69e779
 
 # Used for Taskcluster auth
-taskcluster==0.3.4 --hash sha256:d4fe5e2a44fe27e195b92830ece0a6eb9eb7ad9dc556a0cb16f6f2a6429f1b65
\ No newline at end of file
+taskcluster==0.3.4 --hash sha256:d4fe5e2a44fe27e195b92830ece0a6eb9eb7ad9dc556a0cb16f6f2a6429f1b65
+
+# required by seta
+redo==1.5 --hash=sha256:ad8bdd4c6c9a2b4e2bf35f0aade6a0f24edb889c06d90fe5b5d5203acde865d8
```

Comment:
It seems like this is only used to retry fetching things with requests. I think it would be better to use the celery task retry logic we already have (https://github.com/mozilla/treeherder/blob/608afa4ebd349a7cbf36d5ae93f8d7409782d57d/treeherder/workers/task.py#L12) rather than adding another dependency?

If we do end up using redo, we should probably use the latest version (1.6) :-)